docs/core/deploying/deploy-with-cli.md

```diff
@@ -56,7 +56,7 @@ More information about platform-specific binaries is covered in the [Framework-d
 
 You can use the following app to explore the publishing commands. The app is created by running the following commands in your terminal:
 
-```dotnetcli
+```console
```

Comment:
I think dotnetcli is more suitable since the command `dotnet new console` exists.

Reply:
Oh, I see now the PR was **before** dotnetcli gets supported. I think these should be changed back to dotnetcli now.

Reply:
@Youssef1313 did you get PRs approved using `dotnetcli`
I had no news at all of @mairaw about this

Reply:
You can use dotnetcli now @nemrism